### ISSUE TITLE
Add support for Raspberry Pi 3 B+

### ISSUE THREAD

**Issue by delidana** [alphacharlie/RasPwn#1]
Hi,
Do you have plans to add support for Raspberry Pi 3 B+ ?. I just tried your image and it does not not boot. I'm only getting the rainbow colored screen and nothing more.

**Comment by alphacharlie** [comment]
Yes I am working on a newer version for the 3B+. It will have some differences from the existing image. It will still contain the Raspwn web app playground but is based upon a newer version of Rasbpian. (So the OS/GLibC/kernel stuff will be up-to-date. )

Most of the people who I have talked to who use Raspwn are primarily using the web playground and anyone who wants to play with Spectre, Meltdown or other OS/glibc bugs can still load the current version on an older pi.

thanks for your interest!

**Comment by gunnar7680** [comment]
Any idea on a time frame for 3B+ support?

**Comment by robbelouwet** [comment]
Just bought an RPI 3B+ just to find out that 3B is supported but not 3B+ :/, Any updates @alphacharlie  ?

**Comment by CyberBitz** [comment]
I am waiting for an image as well.

**Comment by othornew** [comment]
I'd be curious to find out as well.

**Comment by feerbau** [comment]
Hello, is there any news on the Raspwn for the Raspberry Pi 3B+? @alphacharlie

**Comment by zisek** [comment]
Hello alphacharlie, I would like to also vote for RasPwn fro Pi 3B+. Is there any chance to get a version today? Thank you very much for your work!!! Martin

**Comment by langdaleweb** [comment]
Not sure if this project is still going but I’d be interested in a 3B+ compatible version too. Thanks!

**Comment by ghost** [comment]
Hey It is 2020 Now When Are You Planning On Releasing The New Version OF The Image We Are All Waiting

**Comment by Netcandy-dev** [comment]
It's 2022 already ;)  I can confirm the current image still does not work on a RPI 3B+.
Simply swapping out everything in the '0.fat' dir in the image (use 7-zip on Windows) for all the stuff in a regular Raspberry OS image but conserving the 'commandline.txt' file gets you past the rainbow screen to a working Linux, but the default kernel lacks some options that are needed by RasPwn (cgroups maybe?). Didn't have time to investigate further.

**Comment by clucode** [comment]
@Netcandy-dev what os version were you using? Do you think there would be a difference between the lite and full versions of the os, re: missing dependencies?
